src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -896,14 +899,11 @@ def perturb_individual_age(self, pop: pd.DataFrame) -> pd.Series:
                 ~negative_ages_mask & to_shift
             ]
             to_shift = negative_ages_mask
-            if i == (max_iterations - 1) and len(to_shift) > 0:
-                print(
-                    "Maximum iterations of resampling of age perturbation reach.  The number of simulants who's age"
-                    "was not perturbed is",
-                    len(to_shift),
+            if i == (max_iterations - 1) and to_shift.sum() > 0:
```

Comment:
Oh note, you can do this logging after the loop completes. Then the `if` check can just be
```
if to_shift.sum() > 0:
    ...
```